src/main/java/codeu/model/data/Message.java

```diff
@@ -26,8 +27,9 @@
   private final UUID id;
   private final UUID conversation;
   private final UUID author;
-  private final String content;
+  private String content;
```

Comment:
In the event where we might want to see a message history, it might be better to keep this field final and instead create a new message and hot swap the ID in the conversation.